pages/renew.tsx

```diff
@@ -85,6 +86,33 @@ export default function Renew() {
       !doctorPostalCode ||
       !doctorPhoneNumber);
 
+  const shopifyCheckout = async () => {
+    /* Setup Shopify Checkout on success. */
+    const client = Client.buildClient({
+      domain: 'poppy-hazel.myshopify.com',
+      storefrontAccessToken: '64b319dde0d72cf6491db234e9a6b3e4',
+    });
+
+    // Product id can be found when viewing URL in Shopify admin (e.g poppy-hazel.myshopify.com/admin/products/6570386915350).
+    // Shopify SDK only accepts encoded product id.
+    const productId = 'gid://shopify/Product/6570386915350';
```

Comment:
Let's put this in an environment variable as well. That way we can mock products as well and have different product IDs in different environments